inspirehep/modules/migrator/tasks.py

```diff
@@ -356,3 +359,72 @@ def migrate_and_insert_record(raw_record):
         prod_record.valid = True
         db.session.merge(prod_record)
         return record
+
+
+@shared_task(ignore_result=True)
+def sync_legacy_claims():
+    """Task to import claims from legacy."""
+    redis_url = current_app.config.get('CACHE_REDIS_URL')
+    r = StrictRedis.from_url(redis_url)
+    lock = Lock(r, 'import_legacy_claims', expire=120, auto_renewal=True)
+    if lock.acquire(blocking=False):
+        try:
+            while r.llen('legacy_claims'):
+                claim = loads(r.lrange('legacy_claims', 0, 1)[0])
+                apply_claim(*claim)
+                r.lpop('legacy_claims')
+        finally:
+            lock.release()
+    else:
+        logger.info('Import_legacy_claims already executed. Skipping.')
+
+
+def apply_claim(inspire_bai, author_recid, lit_recid, signature, flag):
+    """Import claims from legacy and apply them to inspire-next records.
+
+    Args:
+        inspire_bai (string): author profile identifier as BAI
+        author_recid (int): Inspire author record identifier
+        lit_recid (string): Inspire literature record identifier
+        signature (string): name as written on the paper
+        flag (string): record can be ignored if flag == -2 (claim reject)
+    """
+    if flag == -2:
+        return
+
+    try:
+        lit_pid = PersistentIdentifier.get('lit', lit_recid)
+        lit_record = InspireRecord.get_record(lit_pid.object_uuid)
+        for author in get_value(lit_record, 'authors', []):
```

Comment:
`get_value` is useful only when you do chained `get`s. In this case it's faster to do:
```python
for author in record.get('authors', []):
```